frontend/src/theme/primitives/typography.ts

```diff
@@ -0,0 +1,39 @@
+export const fonts = {
+  sansSerif: "IBM Plex Sans, sans-serif",
+  mono: "IBM Plex Mono, monospace",
+}
+
+export const fontSizes = {
+  xxs: "0.375rem",
+  xs: "0.5rem",
+  sm: "0.75rem",
+  smDefault: "0.8rem",
+  md: "1rem",
+  lg: "1.25rem",
+  xl: "1.5rem",
+  xxl: "1.75rem",
+  xxxl: "2.25rem",
+  xxxxl: "3.125rem",
```

Comment:
might just need to clean up our reboot but possible our font sizes could be all over the place but this looks like a good definition to conform to!

Reply:
Yea. I think that's the "goal". And as conversion happens if numbers are off. To be fair, I did search around to find out what we do use.